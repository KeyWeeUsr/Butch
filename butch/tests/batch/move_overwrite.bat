:: run from test.py cwd assuming repo root
cd butch
cd tests
echo %errorlevel%

type source\file1
type source\file2
type dest\file1
type dest\file2

move /Y source\* dest
echo %errorlevel%
type dest\file1
type dest\file2

rmdir source
echo %errorlevel%
del dest\file1
echo %errorlevel%
del dest\file2
echo %errorlevel%
rmdir dest
echo %errorlevel%
cd ..
cd ..
