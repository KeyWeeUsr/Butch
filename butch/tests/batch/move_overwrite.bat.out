0
source-hello
source-world
dest-hello
dest-world
0
source-hello
source-world
0
0
0
0
